projects/1.18-fabric/assets/enhanced-searchability/enhancedsearchability/lang/zh_cn.json

```
@@ -0,0 +1,7 @@
+{
+  "enhancedsearchability.searchbox": "搜索...",

```

```suggestion
  "enhancedsearchability.searchbox": "搜索……",
```
居然还能看到半角省略号
这都说过多少回了